### PR TITLE
Add pysumma

### PR DESCRIPTION
**Pull request by arbennett**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble be please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by bartnijssen** [comment]
I agree to be listed as a maintainer for this package.

File: recipes/pysumma/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+test:
+  imports:
+    - pysumma
+
+about:
+  home: https://github.com/UW-Hydro/pysumma
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/pysumma/meta.yaml:48]: add comment:

```yaml
# license file manually packaged. See https://github.com/UW-Hydro/pysumma/pull/137
license_file: LICENSE
```

I've opened a PR upstream to fix having to manually package the license file (https://github.com/UW-Hydro/pysumma/pull/137). The comment in the recipe will be a visual reminder to check the status when you update the feedstock for future releases. Once the license file is included in the source distribution, you should delete the comment and remove the manually packaged copy.

File: recipes/pysumma/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  url: https://pypi.io/packages/source/p/pysumma/pysumma-{{ version }}.tar.gz
+  sha256: d55e5c372bf9d9b196bfbb591f0bfeac1af51eeb1f58d5b6d95403786542b067
+  
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win]
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/pysumma/meta.yaml:17]: change `build` to `host`

File: recipes/pysumma/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  name: pysumma
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pysumma/pysumma-{{ version }}.tar.gz
+  sha256: d55e5c372bf9d9b196bfbb591f0bfeac1af51eeb1f58d5b6d95403786542b067
+  
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (synapticarbors) [inline recipes/pysumma/meta.yaml:13]: `script: "{{ PYTHON }} -m pip install . -vv"`

Comment (arbennett) [reply recipes/pysumma/meta.yaml]: Updated

File: recipes/pysumma/meta.yaml (hunk 2 above)
Comment (arbennett) [reply recipes/pysumma/meta.yaml]: Updated

File: recipes/pysumma/meta.yaml (hunk 1 above)
Comment (arbennett) [reply recipes/pysumma/meta.yaml:49]: Updated

**Pull request merged**